frc46_token/src/token/mod.rs

```diff
@@ -859,40 +857,46 @@ mod test {
             )
             .unwrap();
         token.flush().unwrap();
-        hook.call(token.msg()).unwrap();
+        hook.call(&token.runtime).unwrap();
 
         assert_eq!(token.total_supply(), TokenAmount::from_atto(100));
 
         // flush token to blockstore
         let cid = token.flush().unwrap();
 
         // the returned cid can be used to reference the same token state
-        let mut state = Token::<_, FakeMessenger>::load_state(&bs, &cid).unwrap();
-        let token2 =
-            Token::wrap(&bs, FakeMessenger::new(TOKEN_ACTOR.id().unwrap(), 6), 1, &mut state);
+        let helper = ActorRuntime {
+            blockstore: token.runtime.blockstore,
+            syscalls: FakeSyscalls::default(),
+        };
```

Comment:
This is a bit unfortunate - the helper is consumed by `wrap`? This will probably strike in actor users outside tests, too.
- It may be appropriate to hold only a reference (w lifetime) to the helper in the token.
- Else it may be appropriate to add a clone() or similar method to the helper to create another one backed by the same things (requiring the syscalls/blockstore to be `+Clone`)

Reply:
though the helper is consumed, it can be (re)accessed via `pub fn runtime(&self) -> ActorRuntime<S, BS>` 

this follows the same ownership and reference structure as TokenState